ui/apps/platform/src/Containers/NetworkGraph/utils/simulatorUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { NetworkPolicyModification } from 'Containers/Network/networkTypes';
+
+export function getDisplayYAMLFromNetworkPolicyModification(
+    modification: NetworkPolicyModification | null
+): string {
+    if (!modification) {
+        return '';
+    }
+    const { applyYaml, toDelete } = modification || {};
+    const shouldDelete = toDelete && toDelete.length > 0;
+    const showApplyYaml = applyYaml && applyYaml.length >= 2;
+
+    // Format toDelete portion of YAML.
+    let toDeleteSection;
+    if (shouldDelete && toDelete) {
+        toDeleteSection = toDelete
+            .map((entry) => `# kubectl -n ${entry.namespace} delete networkpolicy ${entry.name}`)
+            .join('\n');
+    }
```

Comment:
Actually, you can collapse this into an AND to avoid the `let`.

```suggestion
    const toDeleteSection =shouldDelete && toDelete
            .map((entry) => `# kubectl -n ${entry.namespace} delete networkpolicy ${entry.name}`)
            .join('\n');
    }
```